index.html

```diff
@@ -1,12 +1,125 @@
-<!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Нёрдс</title>
-  </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
-
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+   <title>Дизайн-студия «nёrd's»</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <a>
+        <img src="img/index-logo.svg" width="159.87px" height="65.02px" alt="Логотип дизайн-студии «nёrd's»">
+      </a>
+      <ul>
+        <li><a href="blank.html">Студия</a></li>
+        <li><a href="blank.html">Клиенты</a></li>
+        <li><a href="catalog.html">Магазин</a></li>
+        <li><a href="blank.html">Контакты</a></li>
+        <a href="blank.html">Корзина</a>
+      </ul>
+    </nav>
+  </header>
+  <main>
+          <h1>Дизайн-студия «nёrd's»</h1>
+            <!-- h1 визуально будет скрыт,необх для чтения поисковиками и скрин ридерами -->
+        <section>
+          <h2>Долго, дорого, скурпулезно.</h2>

```

Comment:
```suggestion
          <h2>Долго, дорого, скрупулезно.</h2>

```
Текст лучше копировать из макета, так вероятность ошибки будет значительно ниже, чем при наборе вручную. Кроме того такой подходит сэкономит ваше время.